ocrd_tesserocr/segment_region.py

```diff
@@ -219,9 +219,15 @@ def _process_page(self, it, page, page_image, page_coords, page_id):
             else:
                 polygon = polygon_from_x0y0x1y1(bbox)
             polygon = coordinates_for_segment(polygon, page_image, page_coords)
-            polygon = polygon_for_parent(polygon, page)
+            polygon2 = polygon_for_parent(polygon, page)
+            if polygon2 is not None:
+                polygon = polygon2
             points = points_from_polygon(polygon)
             coords = CoordsType(points=points)
+            if polygon2 is None:
```

Comment:
Why the `is None` check after the two lines before? Can't this be combined with the `if polygon2 is not None` check above?

Reply:
Because we also want to generate an informative log message (which needs `points`).

Reply:
I see, makes sense.